content/300-guides/200-deployment/04-deploying-to-azure-functions.mdx

```diff
@@ -0,0 +1,483 @@
+
+If the request succeeds, you should see the created user object returned:
+
+```json
+{
+  "createdAt": "2021-03-02T14:48:15.746Z",
+  "email": "alice@prisma.io",
+  "id": 1,
+  "name": "Alice"
+}
+```
+
+The files associated with the function can be found in the `CreateUser` folder, which contains two files:
+
+- `function.json`: Function configuration, e.g. HTTP method, path, and return value
+- `index.js`: The function handler where Prisma Client is used to create the user in the Azure SQL database
+
+Now, try creating a _post_ associated with the user you just created with the following comand:
+
+```terminal copy
+curl --request POST --data '{"title":"Prisma with Azure","content":"","authorEmail":"alice@prisma.io"}' https://FUNCTION_APP_NAME.azurewebsites.net/api/post
+```
+
+If the request succeeds, you should see the created _post_ object returned:
+
+```json
+{
+  "id": 1,
+  "createdAt": "2021-03-02T17:09:53.160Z",
+  "updatedAt": "2021-03-02T17:09:53.161Z",
+  "title": "Prisma with Azure",
+  "content": "",
+  "published": false,
+  "authorId": 1
+}
+```
+
+To update the `published` field of the post, make the following request:
+
+```terminal copy
+curl --request PUT https://FUNCTION_APP_NAME.azurewebsites.net/api/publish/1
+```
+
+If the request succeeds, you should see the updated _post_ object:
+
+```json
+{
+  "authorId": 1,
+  "content": "",
+  "createdAt": "2021-03-02T17:09:53.160Z",
+  "id": 1,
+  "published": true,
+  "title": "Prisma with Azure",
+  "updatedAt": "2021-03-03T10:07:11.047Z"
+}
+```
+
+Finally, to test the _feed_ endpoint, make the following request:
+
+```terminal copy
+curl https://FUNCTION_APP_NAME.azurewebsites.net/api/feed
+```
+
+If the request succeeds, you should see the post you created and the related author:
+
+```json
+[
+  {
+    "author": {
+      "createdAt": "2021-03-02T14:48:15.746Z",
+      "email": "alice@prisma.io",
+      "id": 1,
+      "name": "Alice"
+    },
+    "authorId": 1,
+    "content": "",
+    "createdAt": "2021-03-02T17:09:53.160Z",
+    "id": 1,
+    "published": true,
+    "title": "Prisma with Azure",
+    "updatedAt": "2021-03-03T10:07:11.047Z"
+  }
+]
+```
+
+## Developing and debugging the functions locally
+
+When implementing Azure Functions, you can also start a local development environment using the Azure Functions Core tools' function runtime. That way, you can test and debug the implementation of the functions locally.
+
+To launch the functions runtime, run the following command:
+
+```terminal copy
+npx func start
+```
+
+The command starts a local server and allows you to call any of the functions in the project.
+
+You can inject environment variables into the functions by adding them to the `Values` object in the `local.settings.json` file at the root of the project. You should, however, avoid adding secrets to that file as it's typically committed to the repository.
```

Comment:
When using VS Code Azure Functions extension, it generates a `.gitignore` file in which the `local.settings.json` is added to it.

Reply:
This makes me wonder if we should remove it from the example repo. The problem then is that the user needs to create it.

Reply:
We should. We can include the instructions in the README and the reason why they're doing that. 
I can help out with that if you'd like.

Reply:
Ok I removed the `local.settings.json` from the example, added a step to add the config and removed this sentence about it getting committed to the repo. 

Let me know what you think :)